polaris-react/src/components/Filters/components/FiltersBar/FiltersBar.tsx

```diff
@@ -162,8 +160,6 @@ export function FiltersBar({
 
   const hasOneOrMorePinnedFilters = pinnedFilters.length >= 1;
 
-  const labelVariant = mdDown ? 'bodyMd' : 'bodySm';
```

Comment:
Are we okay with removing this font size swap at breakpoint?

Reply:
@sam-b-rose No issues from our side with this change if it's the standardised way moving forward.